runtime/vm/JFRChunkWriter.hpp

```diff
@@ -823,6 +833,8 @@ class VM_JFRChunkWriter {
 
 		requiredBufferSize += (_constantPoolTypes.getThreadStatisticsCount() * THREAD_STATISTICS_EVENT_SIZE);
 
+		requiredBufferSize += THREAD_DUMP_EVENT_SIZE_PER_THREAD * _vm->peakThreadCount;
+
```

Comment:
While you're here, please remove the parentheses around the right-hand side of the statement on line 834 for consistency.

Also for consistency, please put the constant on line 836 on the right:
```c
		requiredBufferSize += _constantPoolTypes.getThreadStatisticsCount() * THREAD_STATISTICS_EVENT_SIZE;

		requiredBufferSize += _vm->peakThreadCount * THREAD_DUMP_EVENT_SIZE_PER_THREAD;
```